tilavarauspalvelu/tasks.py

```diff
@@ -711,6 +712,92 @@ def anonymize_old_users_task() -> None:
     retry_backoff=True,
 )
 def delete_pindora_reservation(reservation_uuid: str) -> None:
+    """
+    Task that can be used to retry a Pindora reservation deletion if it fails
+    in the endpoint. This should only be called if the access code is know to
+    be inactive, since this task may also fail to delete the reservation if
+    Pindora is down for an extended period of time.
+    """
     from tilavarauspalvelu.integrations.keyless_entry import PindoraClient
 
     PindoraClient.delete_reservation(reservation=uuid.UUID(reservation_uuid))
+
+
+@app.task(name="create_missing_pindora_reservations")
+def create_missing_pindora_reservations() -> None:
```

Comment:
Thinking maybe these tasks can time out if there is a lot of reservations to be updated? In that case nothing will move forward. Maybe we can just update them in the loop?

Reply:
Changed behavior to update in the loop, added comment.